ui/components/MesheryPerformance/index.js

```diff
@@ -914,7 +914,7 @@ class MesheryPerformanceComponent extends React.Component {
                         <TextField
                           id="cookies"
                           name="cookies"

```

Comment:
@adithyaakrishna please consider changing this and id to "request body"

Reply:
Okay Sure